### PR TITLE
[TR#183] Dropbox Component

### PR DESCRIPTION
**Pull request by duskcloudxu**
#### Please check if the PR fulfills these requirements

- [x] The changes don't come from your master branch.

---

#### What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)
Added a dropbox component based on react-native-picker-select.

#### Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)
No
#### Other information
for test, there is an instance of dropboxinput under registrationScreen, you could access it by clicking register in login page.

---

#### Images (before/ after screenshots, interaction GIFs, ...)
Kapture 2020-06-09 at 22 54 49

### CODE REVIEW

File: src/elements/FormTextInput/DropdownInput/DropdownInput.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+						right: 2,
+					},
+					placeholder: {
+						color: colors.NAVY_BLUE,
+						fontSize: 16,
+					},
+				}
+
+			}
+			placeholder={{
```
Comment (emarti43) [inline src/elements/FormTextInput/DropdownInput/DropdownInput.tsx]: The placeholder here creates another dropdown item, creating 2 `"WA"` selections. I believe `DonorRegistrationScreen` passes `"WA"` as an initial value and can be passed down to `DropdownInput`

**Review by emarti43 — COMMENTED** [review]
This looks good. There is a small thing I noticed, but besides that, it's working on my end 👍 .

File: src/elements/FormTextInput/DropdownInput/DropdownInput.tsx (hunk 1 above)
Comment (duskcloudxu) [reply src/elements/FormTextInput/DropdownInput/DropdownInput.tsx]: Thanks for reviewing! Are you testing on android? It works fine on IOS...

Comment (duskcloudxu) [reply src/elements/FormTextInput/DropdownInput/DropdownInput.tsx]: Oh I found it out, nice catch! I'll update and push modification late.

**Review by emarti43 — APPROVED** [review]
works on android 👍

**Pull request merged**